### PR TITLE
Paris - Christina & Christophe

### PR DESCRIPTION
**Pull request by grosbegue**
We lost some time finding out what was expected of us, but we grasped most of the exercice by now

@cibailey

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
+      integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <div class="container">
```
Comment (ta-web-paris) [inline index.html]: The `<div class="container">` should be inside your `<div class="header">`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      <div class="header">
```
Comment (ta-web-paris) [inline index.html]: You can remplace `<div class="header">` by `<header>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        <div class="row">
+          <div class="col-4"><h1 class="logo">REVERRA</h1></div>
+          <div class="col-8">
+            <nav class="navbar navbar-expand-lg navbar-light bg-light">
```
Comment (ta-web-paris) [inline index.html]: Your nav have not burger menu for responsive :) 
The correction of it is : 

```
            <nav class="navbar navbar-expand-lg navbar-light bg-light">
                
                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                  <span class="navbar-toggler-icon"></span>
                </button>
                <div class="collapse navbar-collapse" id="navbarNav">
                  <ul class="navbar-nav">
                    <li class="nav-item active">
                      <a class="nav-link" href="#">About <span class="sr-only">(current)</span></a>
                    </li>
                    <li class="nav-item">
                      <a class="nav-link" href="#">Sample Pages</a>
                    </li>
                    <li class="nav-item">
                      <a class="nav-link" href="#">Portfolio</a>
                    </li>
                    <li class="nav-item">
                      <a class="nav-link disabled" href="#">Blog</a>
                    </li>
                    <li class="nav-item">
                      <a class="nav-link disabled" href="#">Parent Pages</a>
                    </li>
                  </ul>
                </div>
              </nav>
```

**Comment by ta-web-paris** [comment]
I know that you had not much time to do this exercise, but I really encourage you to finish it by yourself this week(-end).
You’ll need to use bootstrap during your projects so that’s better to practice before to not loose time with it after :) 
If you’re looking for the correction, here it is : https://github.com/ironhack-dev-squad-107/html-css-bootstrap
Good luck :)

**Comment by CIBailey** [comment]
Thanks Laura! I plan on doing that :)

Sent from my iPhone

On 16 Jan 2019, at 11:39, TA paris <notifications@github.com<mailto:notifications@github.com>> wrote:


I know that you had not much time to do this exercise, but I really encourage you to finish it by yourself this week(-end).
You’ll need to use bootstrap during your projects so that’s better to practice before to not loose time with it after :)
If you’re looking for the correction, here it is : https://github.com/ironhack-dev-squad-107/html-css-bootstrap
Good luck :)

**Pull request closed without merging**